kubernetes/kubernetes_test.go

```diff
@@ -379,6 +410,77 @@ func TestCollectMetrics(t *testing.T) {
 			}
 		}
 	})
+	t.Run("Ensure collection occurs with parseMetrics is disabled"+
+		" ensure sensitive data is not stripped", func(t *testing.T) {
+
+		filepath.Walk(ka.msExportDirectory.Name(), func(path string, info os.FileInfo, err error) error {
+			if strings.Contains(info.Name(), "jsonl") {
+				if strings.Contains(info.Name(), "pods") {
+					// check if secrets were not stripped from pods if parseMetrics is false
+					in, _ := os.ReadFile(path)
```

Comment:
Not a huge deal since it's in a test file, but this could be bumped up to above the first if and the passed in error should be checked along with the return of os.ReadFile

Reply:
thanks, made the changes